docs/customize/administration_interface.md

```diff
@@ -0,0 +1,398 @@
+class OaiPmhDetailView(AdminResourceDetailView):
+    """Configuration for OAI-PMH sets detail view."""
+
+    url = "/oai-pmh/<pid_value>"
+    api_endpoint = "/oaipmh/sets"
+    search_request_headers = {"Accept": "application/json"}
+    name = "OAI-PMH details"
+    resource_config = "oaipmh_server_resource"
+    title = "OAI-PMH Details"
+
+    template = "invenio_rdm_records/oai-details.html"
+    display_delete = True
+    display_edit = True
+
+    list_view_name = "OAI-PMH"
+    pid_path = "id"
+
+    item_field_list = {
+        "name": {"text": "Set name", "order": 1},
+        "spec": {"text": "Set spec", "order": 2},
+        "search_pattern": {"text": "Search query", "order": 3},
+        "created": {"text": "Created", "order": 4},
+        "updated": {"text": "Updated", "order": 5},
+    }
+
+```
+
+#### Views registration
+
+In order to render views in `invenio-administration`, they must be registered from the module on which they are implemented.
+
+A typical flow of actions to register views for a module is as follows:
+
+1. Create views in the module.
+2. Configure views to match your needs.
+3. Register views in `invenio-administration` entry point group.
+
+In OAI-PMH sets example, views are registered in `invenio-administration` as individual entry points from `invenio-rdm-records`.
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_rdm_records_oai_list = invenio_rdm_records.administration.views.oai:OaiPmhListView
+    invenio_rdm_records_oai_edit = invenio_rdm_records.administration.views.oai:OaiPmhEditView
+    invenio_rdm_records_oai_create = invenio_rdm_records.administration.views.oai:OaiPmhCreateView
+    invenio_rdm_records_details = invenio_rdm_records.administration.views.oai:OaiPmhDetailView
+```
+
+### Create custom view
+
+A custom view can be created by inheriting directly from `AdminView`.
+#### Usage
+
+```python
+from invenio_administration.views.base import AdminView
+
+class MyCustomView(AdminView):
+    """My custom view."""
+
+    extension_name = None
+    category = None
+    template = "invenio_administration/index.html"
+    url = None
+    menu_label = None
+    order = None
+    icon = None
+```
+
+- **extension_name** `String` optional: Flask extension of the view. Default is built with the name of the entry point
+- **category** `String` optional: Menu entry that contains the view. Default is None.
+- **template** `String` optional: Path to the custom template.
+- **url** `String`  optional: URL used to route the view. Default is built with the name of the custom view.
+- **menu_label** `String` optional: Label that will be displayed in the menu.
+- **order** `String` optional: Order of the menu entry.
+- **icon** `String` optional: The optional icon that the menu entry might have. The value should be one of the [Semantic UI icons](https://react.semantic-ui.com/elements/icon/).
+
+### Customisation: dashboard view
```

Comment:
At CERN we favor UK eng, but I think since the docs have sections in US (e.g. customiZe) we can default to US? or change all to UK.